src/Classes/PolygonMask.ts

```diff
@@ -34,6 +34,9 @@ export class PolygonMask {
     public path?: SVGPathElement;
 
     private readonly container: Container;
+    private readonly path2DSupported = !!(window as any)["Path2D"];
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Type declaration of 'any' loses type-safety. Consider replacing it with a more precise type.](https://app.codacy.com/manual/ar3s/tsparticles/pullRequest?prid=5182069)